klarrio-build.sh

```diff
@@ -0,0 +1,389 @@
+#!/bin/bash
+
+# Expects a 'klarrio-build.json' file to be present in the repository, with the following content:
```

Comment:
won't it be better to put it in https://github.com/Klarrio/dsh-tool-utils/tree/master/scripts to avoid multiple copies of it?

Reply:
I think doing that would result to the comment made by Dominique, where people can't be bothered to use the script or simply don't know about it because it isn't included in the repo.

Reply:
Yeah that could very well be